README.md

```diff
@@ -6,15 +11,256 @@
 
 Official Customer.io SDK for iOS
 
-**Note: This project is a work in progress and is not yet ready for general availability. If you are interested in being an early user of this SDK for your iOS app, send us a message at `product@customer.io` saying you would like to try out our iOS SDK. We will work with you to get setup!**
+**This project is a work in progress and is not yet ready for general availability. If you are interested in being an early user of this SDK for your iOS app, send us a message at `product@customer.io` saying you would like to try out our iOS SDK. We will work with you to get setup!**
+
+Notes:
+1. The SDK has been tested on iOS devices. The SDK may work on other Apple devices such as macOS, tvOS, and watchOS but official support and official testing has not been done on these devices. 
+2. The SDK is in `alpha` phase at the moment. It is available to use but note that the API will have breaking changes introduced and issues may occur. 
```

Comment:
Could we include a list of the available features + upcoming additions here?  That way customers know what to expect and have a preview of what's to come.